.github/workflows/check-providers.yml

```diff
@@ -0,0 +1,40 @@
+name: Check Providers
+
+on:
+  pull_request:
```

Comment:
Currently this workflow will run:

1. when a pull request is opened - not sure if we want this to run on every push (?)
2. on a cron schedule at 12:00

Reply:
You can do it in PRs that makes any changes to `chains/` as in https://github.com/api3dao/airnode/blob/master/.github/workflows/protocol-verify.yml#L7